index.html

```diff
@@ -37,8 +37,128 @@ <h1>I'm Sohail Glad to see you!</h1>
+                <img src="./images/Snapshoot Portfolio.png" alt="">
+                <div class="card-work work">
+                    <h3>Multi-Post <br> Stories</h3>
+                    <ul class="proj">
+                        <li><a href="#">CANOPY</a></li>
+                        <li><div class="dot"></div></li>
+                        <li>Back End Dev</li>
+                        <li><div class="dot"></div></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="pro">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="language">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
+                </div>
+            </div>
+            <div class="works">
+                <img src="./images/Snapshoot Portfolio-1.png" alt="">
+                <div class="card-work work">
+                    <h3>Tonic</h3>
+                    <ul class="proj">
+                        <li><a href="#">CANOPY</a></li>
+                        <li><div class="dot"></div></li>
+                        <li>Back End Dev</li>
+                        <li><div class="dot"></div></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="pro">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="language">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
+                </div>
+            </div>
+            <div class="works">
+                <img src="./images/Snapshoot Portfolio-2.png" alt="">
+                <div class="card-work work">
+                    <h3>Multi-Post <br> Stories</h3>
+                    <ul class="proj">
+                        <li><a href="#">CANOPY</a></li>
+                        <li><div class="dot"></div></li>
+                        <li>Back End Dev</li>
+                        <li><div class="dot"></div></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="pro">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="language">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
+                </div>
+            </div>
+        </div>
+<!-- ------------------------------------------------about me--------------------------------         -->
+        <div class="details">
```

Comment:
- [ ] According to the design, I suggest to put a border-top-right-radius  to the about section, as shown here:

  - Your design:
![rRadius](https://user-images.githubusercontent.com/97834160/205107266-f2ca9cfe-6f97-4004-9fdb-42ee3ac20b57.png)

  - Figma design:
![rightRadius](https://user-images.githubusercontent.com/97834160/205107314-4995dc20-738c-4e46-be06-fdc70c452f28.png)